polkadot-parachains/encointer-runtime/src/xcm_config.rs

```diff
@@ -218,6 +218,8 @@ impl pallet_xcm::Config for Runtime {
 	type ReachableDest = ReachableDest;
 	type AdminOrigin = EnsureRoot<AccountId>;
 	type UniversalLocation = UniversalLocation;
+	type MaxRemoteLockConsumers = ConstU32<0>;
+	type RemoteLockConsumerIdentifier = ();
```

Comment:
I assume that these are aligned with statemin*?

Reply:
Yes. Copy paste